src/components/carousel/index.tsx

```diff
@@ -520,7 +520,7 @@ class Carousel extends Component<CarouselProps, CarouselState> {
           pagingEnabled={this.shouldEnablePagination()}
           snapToOffsets={snapToOffsets}
           contentOffset={contentOffset}
-          // onContentSizeChange={this.onContentSizeChange}
+          onContentSizeChange={this.onContentSizeChange}
```

Comment:
I saw that they commented that this might not be needed in their pr. It seems to fix the starting page problem now.
Regarding the ticket the the PR you mentioned fixed. Its not reproduced after this change.

Reply:
I used this fix and tested in on our demo app, in Android phone in Hebrew.
The index does fixed in RTL, but the initial index passed is 0 and on RTL it's starting on 6, does this PR should solve this also ?

Reply:
Run the app on Android emulator, make sure to force RTL.
In the carousel screen (in the private repo) check the `INITIAL_PAGE` value, the `currentPage` in the state is the reversed value of the `INITIAL_PAGE`.